site/src/pages/release-notes/1.29.0.mdx

```diff
@@ -0,0 +1,248 @@
+---
+date: 2024-06-05
+---
+
+## 🌟 New features
+
+- You can now use an <type://AsyncLoader> to load a value atomically and asynchronously. #5506 #5590
+  ```java
+  Function<String, CompletableFuture<String>> loader = cache -> {
+    ResponseEntity<String> response =
+      client.prepare().get("/api/v1/items").asString().execute();
+      return response.thenApply(res -> res.content());
+  };
+
+  AsyncLoader<String> asyncLoader =
+    AsyncLoader
+      .builder(loader)
+      .expireAfterLoad(Duration.ofSeconds(60))
+      .build();
+  asyncLoader.get().join();
+  ```
+- Armeria [HTTP/JSON to GRPC transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding) service now
+  supports [Custom methods](https://cloud.google.com/apis/design/custom_methods). #5613
+  ```protobuf
+  rpc Watch(WatchRequest) returns (WatchResponse) {
+    option (google.api.http) = {
+      post: "/v1:watch" // 👈👈👈
+      body: "*"
+    };
+  }
+  ```
+- You can now specify when to remove multipart temporary files using <type://MultipartRemovalStrategy>. #5652 #5653
+  ```java
+  Server
+    .builder()
+    .multipartRemovalStrategy(MultipartRemovalStrategy.ON_RESPONSE_COMPLETION)
+    ...
+  ```
+- A <type://ClientConnectionTimings> now includes the time spent on the TLS handshake. #3647 #5647
+  ```java
+  ClientConnectionTimings timings = ...
+  timings.tlsHandshakeDurationNanos();
+  ```
+- You can now configure <type://TlsEngineType> using <type://ClientFactoryBuilder> or <type://ServerBuilder>. #4962
+  ```java
+  ClientFactory
+    .builder()
+    .tlsEngineType(TlsEngineType.OPENSSL)
+    .build();
+  ```
+- You can now easily find both dynamic and static decorators that handle a request
+  via <type://ServiceRequestContext.findService(Class)>. #5670
+  ```java
+  ServerBuilder sb = ...
+  sb.service("/foo", fooService);
+  ...
+  ServiceRequestContext ctx = ...
+  assert ctx.findService(FooService.class) == fooService;
+  ```
```

Comment:
I feel like this is not the best example. Can you use an example that uses a route decorator? `CorsService` could be looked for.